src/store/reducers/devicesReducer.js

```diff
@@ -81,7 +81,12 @@ const deleteDevicesReducer = (state, { payload }) => {
 };
 
 const insertDevicesReducer = (state, { payload }) => {
-  const { entities: { devices }, result } = normalize(payload, deviceListSchema);
+  const inserted = payload.reduce((acc, payloadItem) => {
```

Comment:
Looks like payload is already an array so using a `map` here should be sufficient.
```js
const inserted = payload.map(device => ({ ...device, showGlimmer: true }));
```